src/test/store/profile-view.test.js

```diff
@@ -235,14 +236,20 @@ describe('getJankMarkersForHeader', function() {
   });
 
   it('will create a jank instance with eventDelay values', function() {
-    const breakingPoint = 70;
-    const eventDelay = [0, 20, 40, 60, breakingPoint, 0, 20, 40];
+    const breakingPoint = 50;
+    // We need to fill the samples before, because the algorithm looks at the
+    // later event delay values and determines the previous event delays from them.
+    const eventDelay = Array(50).fill(0);
```

Comment:
well, I still need to give an eventDelay value to the `getProfileWithEventDelays` function because I want to test a specific event delay value case. I changed the call in the setup function though to use getProfileWithEventDelays though.